Makefile

```diff
@@ -126,7 +126,7 @@ $(ZOT):
 	$(call dlbin,$@,https://github.com/project-zot/zot/releases/download/$(ZOT_VERSION)/zot-linux-amd64-minimal)
 
 $(SKOPEO):
-	@mkdir -p "$(TOOLS_D)/bin"; \
+	@set -e; mkdir -p "$(TOOLS_D)/bin"; \
```

Comment:
This looks fine to me, though could use commands joined with `&&` instead of `set -e`.

Reply:
there really isn't any point in 'set -e' *and* '&&' . they're kind of redundant. no?

Reply:
You're right; I suggested to use `&&` instead since we've discussed the short-comings of `set -e` elsewhere.   Here I think it works fine.